testing/framework/TestCommonTests.py

```diff
@@ -174,12 +176,16 @@ def test___init__(self) -> None:
 
 class banner_TestCase(TestCommonTestCase):
     create_run_env = False
+
     def test_banner(self) -> None:
         """Test banner()"""
         tc = TestCommon.TestCommon(workdir='')
 
         b = tc.banner('xyzzy ')
-        assert b == "xyzzy ==========================================================================", b
+        assert (
```

Comment:
This looks wrong?

Reply:
It's ugly, but not syntactically wrong. We're forcing a very long line by using banner(), and there's no good way to break it with a tool.  I can confirm if asked to do that chunk again, it produces the same result.

Reply:
Hmm. You used black (or ruff) for this?

Reply:
If we set longer (can we) allowable line lengths would some of the ugliness of the reformatting be fixed?

Reply:
Yes - the in-string snippets were also formatted with black (in vim I can select lines and pipe that buffer into an external command). Line length is one we can fiddle - one of the few tunables in black (I think ruff lets you do a few more).  It's just, we don't want to set the overall line length for the project, and if we don't - it will be prettier this time and flip back on next reformat.

Reply:
Hm.. I'm ok with setting a longer than default line length. Almost all displays are wider than high, so longer lines is better than more lines..?

Reply:
> Hm.. I'm ok with setting a longer than default line length. Almost all displays are wider than high, so longer lines is better than more lines..?

Well... I routinely have three 80-column text windows open next to each other, for simultaneous viewing, and often two in an IDE, where all the extra crud, without popping into some weird mode, makes it impossible to fit three. I find it really irritating when lines are longer than 80 because of that and because long lines just seem to make comprehension harder.  I don't even like the black (copied by ruff) default of "88 is little enough over 80 that we'll just let it go".  But that's just one nerd's opinion.

Reply:
I can of course replace the long line of equals with an f-string to produce it - something that's happened in some other places. Then it should all format sanely.

Reply:
Okay, this is done - see if this makes you happier?  Was going to have to force-push anyway, so went ahead and rebased before doing so.